src/core/hash.zig

```diff
@@ -1,12 +1,17 @@
 const std = @import("std");
+const sig = @import("../sig.zig");
+
 const Sha256 = std.crypto.hash.sha2.Sha256;
-const base58 = @import("base58-zig");
 const Allocator = std.mem.Allocator;
 
-pub const HASH_SIZE: usize = 32;
+pub const HASH_SIZE: usize = Hash.size;
```

Comment:
I don't see why `HASH_SIZE` is better than `Hash.size`.

Reply:
HASH_SIZE is already used throughout the codebase and I wasn't trying to change every usage in this pr.

Reply:
ef6543db2bce46db98fefc69cddcfc48d37d4439